README.md

```diff
@@ -14,10 +14,11 @@ This repo has got several branches, illustrating slightly different workflows:
 
 - the `master` branch contains a standard multi app which runs in all engines.
 - the `shotgun_multi_select` branch shows how to get started if you want to create
-  a *shotgun specific* app where you can select multiple objects in ShotGrid and
-  pass this selection into the app.
-- the `shotgun_model_delegate` branch illustrates how to use the ShotGrid MVC model
-  for efficient data retrieval and the toolkit widget delegate system for display of content.
+  a *shotgun specific* app where you can select multiple objects in
```

Comment:
```suggestion
  a *toolkit specific* app where you can select multiple objects in
```

I'm not sure about this. @julien-lang what do you think?

Reply:
I think it should be:
```suggestion
  a *Flow Production Tracking Toolkit specific* app where you can select multiple objects in
```